source/_integrations/plugwise.markdown

```diff
@@ -236,33 +240,42 @@ script:
           preset_mode: asleep
 ```
 
-### Supported devices
+## Supported devices
+
+### Adam
+
+A complete zone control system also known as Adam HA, supporting:
```

Comment:
Add info about the Adam SA (Stand Alone, no ZigBee stick present)  to make a single, basic room thermostat smart?

Reply:
Please crosscheck

Reply:
Is it ok now?